src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img src="images/imac.jpeg"
+        alt="Apple A1419"
+        class="img card__img"
+      >
+      <section class="card__article-information">
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+        <div class="card__stars-reviews">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <p class="card__review">
+            Reviews: 5
+          </p>
+        </div>
```

Comment:
Add space between sibling elements like this (there is no need for space for the list element here, it is more readable this way):
```suggestion
        <div class="card__stars-reviews">
          <ul class="stars stars--4">
            <li class="stars__star"></li>
            <li class="stars__star"></li>
            <li class="stars__star"></li>
            <li class="stars__star"></li>
            <li class="stars__star"></li>
          </ul>
          
          <p class="card__review">
            Reviews: 5
          </p>
        </div>
```

Add empty lines in the whole document for siblings elements